MQTT/MQTTJCAAPI/src/main/java/fish/payara/cloud/connectors/mqtt/api/inbound/MQTTSubscriber.java

```diff
@@ -120,5 +127,27 @@ void close() throws ResourceException {
     public void messageArrived(String topic, MqttMessage mm) throws Exception {
         manager.scheduleWork(new MQTTWork(topic, mm, factory));
     }
+
+    @Override
+    public void connectComplete(boolean reconnect, String serverURI) {
+        if (reconnect) {
+            try {
+                LOGGER.info("Reconnection to " + serverURI + " complete resubscribing for topic " + topic );
```

Comment:
Minor change. String concatenation should be avoided.

Reply:
Done